vpr/src/route/rr_graph.cpp

```diff
@@ -862,15 +861,51 @@ static void load_rr_switch_inf(const int num_arch_switches, const float R_minW_n
             device_ctx.switch_fanin_remap[i_arch_switch][fanin] = i_rr_switch;
 
             load_rr_switch_from_arch_switch(i_arch_switch, i_rr_switch, fanin, R_minW_nmos, R_minW_pmos);
+            //create_rr_switch_from_arch_switch(i_rr_switch,fanin, R_minW_nmos,R_minW_pmos);
```

Comment:
I believe the QoR failures is due to that you are still using the ``load_rr_switch_from_arch_switch`` which modifies the ``rr_segments`` in ``DeviceContext`` rather than the internal data of ``RRGraphBuilder``